src/handlers/imgix.ts

```diff
@@ -39,8 +50,64 @@ const staticImgixClient = shouldCreateImgixClient();
 // TODO: We need to find a suitable upper limit.
 //       This might be conditional based upon either the runtime
 //       hardware or the number of unique tokens a user may have.
-const capacity = 512;
-export const staticSignatureLRU = new LRUCache<string, string>(capacity);
+const capacity = 1024;
+export let staticSignatureLRU: LRUCache<string, string> = new LRUCache(
+  capacity
+);
+
+const maybeReadCacheFromMemory = async (): Promise<
+  LRUCache<string, string>
+> => {
+  try {
+    const cache = new LRUCache<string, string>(capacity);
+    const keys = imgixCacheStorage.getString(ATTRIBUTES.KEYS)?.split(',') ?? [];
+    const values =
+      imgixCacheStorage.getString(ATTRIBUTES.VALUES)?.split(',') ?? [];
+
+    for (let i = 0; i < keys.length; i++) {
+      cache.set(keys[i], values[i]);
+    }
```

Comment:
I would measure if having it as object is faster in general

```
const entires  = cache.entries();
mmkv.set('cache', JSON.stringify(entries));

// later on

const cached = mmkv.getString('cache');
const cachedEntries = JSON.parse(cached);

cache.from(Object.fromEtnries(entries));


```

Reply:
`cache.entries()` returns an iterator and when passed to `JSON.stringify` it turns into `{"done":false}`, so I changed it to reuse existing code for iterating over it.

Saving:

```tsx
const entries = [];

const iterator = staticSignatureLRU.entries();
for (let i = 0; i < staticSignatureLRU.size; i++) {
  const [key, value] = iterator.next().value;
  entries.push([key, value]);
}

imgixCacheStorage.set('cache', JSON.stringify(entries));
```

Reading:
```tsx
const cacheString = imgixCacheStorage.getString('cache') || '';
const entries = JSON.parse(cacheString);
return LRUCache.from(Object.fromEntries(entries), capacity);
```

This is consistently 1.5-2x slower than reading keys and values separately.